chunky/src/java/se/llbit/chunky/ui/render/tabs/LightingTab.java

```diff
@@ -101,7 +106,21 @@ public LightingTab() throws IOException {
 
     enableEmitters.setTooltip(new Tooltip("Allow blocks to emit light based on their material settings."));
     enableEmitters.selectedProperty().addListener(
-      (observable, oldValue, newValue) -> scene.setEmittersEnabled(newValue));
+      (observable, oldValue, newValue) -> {
+        scene.setEmittersEnabled(newValue);
+        emitterSettings.setVisible(newValue);
+        emitterSettings.setManaged(newValue);
+      });
+    boolean showEmitterSettings = scene != null && scene.getEmittersEnabled();
+    emitterSettings.setVisible(showEmitterSettings);
+    emitterSettings.setManaged(showEmitterSettings);
+
+//    fancierTranslucency.selectedProperty()
+//      .addListener((observable, oldValue, newValue) -> {
+//        scene.setFancierTranslucency(newValue);
+//        transmissivityCap.setVisible(newValue);
+//        transmissivityCap.setManaged(newValue);
+//      });
```

Comment:
This seems unrelated?

Reply:
Huh I wonder how that got there, I'll remove it